Assets/Scripts/Inventory/PlayerInventory.cs

```diff
@@ -16,6 +16,10 @@ public class PlayerInventory : MonoBehaviour
 
         private int MaxIngredients = 0;
 
+        // Event to notify when a drink is added or removed
+        public delegate void DrinkChanged();
+        public event DrinkChanged OnDrinkChanged;
```

Comment:
You don't have to make your own type, you can see examples of events in managers, they just use Action<>

Reply:
i believe i have it intergrated, will test before requesting further pr review